components/drone_mobile.json

```diff
@@ -0,0 +1,6 @@
+{	
```

Comment:
Please remove the empty tabs after each line.

Additionally, why did you open a new PR instead of adjusting #419 ?

Reply:
Done. Sorry I just copied and pasted from within the browser. didn't see it had extra spaces.